stages/org.osbuild.anaconda

```diff
@@ -33,24 +56,30 @@ CONFIG = """
 # osbuild customizations
 
 [Anaconda]
-# List of enabled Anaconda DBus modules
-kickstart_modules =
 """
 
 
 def main(tree, options):
-    modules = options["kickstart-modules"]
     product_dir = os.path.join(tree, "etc/anaconda/conf.d")
     os.makedirs(product_dir, exist_ok=True)
 
-    with open(os.path.join(product_dir, "90-osbuild.conf"), "w", encoding="utf8") as f:
-        f.write(CONFIG)
```

Comment:
Am I blind, or did the change drop writing the `CONFIG` to the file?

Reply:
Oof yes, updated.